### ISSUE TITLE
Error, while uploading new design

### ISSUE THREAD

**Issue by karokar** [avocode/avocode#675]
Crashes, everytime at 95%, on specific mockup. 

-----
App v.2.5.2
Source UID: 47988
```
Error: Didn't get expected byte count
expect: 19966
actual: 33278
  at decorate (C:\Users\Maria\AppData\Local\avocode\app-2.5.2\resources\app.asar\node_modules\fstream\lib\abstract.js:67:36)
  at FileReader.Abstract.error (C:\Users\Maria\AppData\Local\avocode\app-2.5.2\resources\app.asar\node_modules\fstream\lib\abstract.js:61:12)
  at ReadStream.<anonymous> (C:\Users\Maria\AppData\Local\avocode\app-2.5.2\resources\app.asar\node_modules\fstream\lib\file-reader.js:66:10)
  at emitNone (events.js:72:20)
  at ReadStream.emit (events.js:163:7)
  at _stream_readable.js:891:16
  at process._tickCallback (node.js:347:13)

```
sometimes get this after that: #507

**Comment by vojtatranta** [comment]
Thank you for your report.
Were are sorry to here that.
We know that some of the designs cause crash on Windows, because they are way too big.
How large is your PSD?

**Comment by karokar** [comment]
About 45 Mb. I have managed to upload it uevetually, after several tries on linux.

**Comment by petrbrzek** [comment]
This should be fixed in 2.8.4. Let us know if the problem persists. I'm closing the issue now.
